### PR TITLE
Css three column layout and new folder for html Nathan Chon

### PR DESCRIPTION
**Pull request by Nathan-Chon**
Screenshot_20230126_062805
I created a new folder for html but its had me add alot of files. sorry!

### CODE REVIEW

File: css-three-column-layout/index.html [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <title>CSS Challenge - Three Column Layout</title>
+  <link rel="stylesheet" href="fonts.css">
+  <link rel="stylesheet" href="reset.css">
+  <link rel="stylesheet" href="styles.css">
+</head>
+<body>
+  <div class="container">
+    <div class="row">
+      <div class="columnone">
```
Comment (robgmerrill) [inline css-three-column-layout/index.html:15]: I'd recommend `.columnone` be renamed something like `.col-whole` or `.col-full` to communicate that this is meant to take the full width.

File: css-three-column-layout/index.html [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+        <h2>Designed for life science. Optimized for SaMD and Digital Health.</h2>
+      </div>
+    </div>
+    <div class="row">
+      <div class="columnthree">
+        <div class="circle">
+          <img src="images/merge.png" alt="merge">
+        </div>
+      </div>
+      <div class="columnthree">
```
Comment (robgmerrill) [inline css-three-column-layout/index.html:25]: I'd recommend that `.columnthree` be rename to `.col-third` to communicate that this is meant to be a third of the width.

**Review by robgmerrill — CHANGES_REQUESTED** [review]
Great start! There are a couple class names that I think could benefit from renaming for clarity of purpose.

**Comment by Nathan-Chon** [comment]
Screenshot_20230126_062805
Revised the class names

**Review by robgmerrill — APPROVED** [review]
👍

**Pull request merged**